bumble/host.py

```diff
@@ -721,6 +721,14 @@ def on_hci_le_advertising_report_event(self, event):
     def on_hci_le_extended_advertising_report_event(self, event):
         self.on_hci_le_advertising_report_event(event)
 
+    def on_hci_le_advertising_set_terminated_event(self, event):
```

Comment:
It would be nice to make the advertiser itself an event emitter, and then have the advertiser emit a 'termination' event.